.yarn/patches/@metamask-assets-controllers-npm-91.0.0-ea998cb0bd.patch

```diff
@@ -0,0 +1,148 @@
++    '0x3e7': null, //'eip155:999/slip44:2457', // HyperEVM - Native symbol: ETH
++    '0x440': null, //'eip155:1088/erc20:0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000', // 'eip155:1088/slip44:XXX', // Metis Andromeda Mainnet (Ethereum L2) - Native symbol: METIS
++    '0x44d': null, //'eip155:1101/slip44:60', // Polygon zkEVM mainnet - Native symbol: ETH
++    '0x504': null, //'eip155:1284/slip44:1284', // Moonbeam - Native symbol: GLMR
++    '0x505': null, //'eip155:1285/slip44:1285', // Moonriver - Native symbol: MOVR
++    '0x531': null, //'eip155:1329/slip44:19000118', // Sei Mainnet - Native symbol: SEI
++    '0x1388': null, //'eip155:5000/erc20:0xdeaddeaddeaddeaddeaddeaddeaddeaddead0000', // 'eip155:5000/slip44:XXX', // Mantle - Native symbol: MNT
++    '0x2105': null, //'eip155:8453/slip44:60', // Base - Native symbol: ETH
++    '0x2710': null, //'eip155:10000/erc20:0x0000000000000000000000000000000000000000', // 'eip155:10000/slip44:145', // Smart Bitcoin Cash - Native symbol: BCH
++    '0xa4b1': null, //'eip155:42161/slip44:60', // Arbitrum One - Native symbol: ETH
++    '0xa4ec': null, //'eip155:42220/slip44:52752', // Celo Mainnet - Native symbol: CELO
++    '0xa516': null, //'eip155:42262/erc20:0x0000000000000000000000000000000000000000', // 'eip155:42262/slip44:474', // Oasis Emerald - Native symbol: ROSE
++    '0xa86a': null, //'eip155:43114/slip44:9005', // Avalanche C-Chain - Native symbol: AVAX
++    '0xe708': null, //'eip155:59144/slip44:60', // Linea Mainnet - Native symbol: ETH
++    '0x13c31': null, //'eip155:81457/erc20:0x0000000000000000000000000000000000000000', // 'eip155:81457/slip44:60', // Blast Mainnet - Native symbol: ETH
++    '0x17dcd': null, //'eip155:97741/erc20:0x0000000000000000000000000000000000000000', // 'eip155:97741/slip44:XXX', // Pepe Unchained Mainnet - Native symbol: PEPU
+     '0x518af': null, // 'eip155:333999/slip44:1997', // Polis Mainnet - Native symbol: POLIS
+-    '0x82750': 'eip155:534352/slip44:60', // Scroll Mainnet - Native symbol: ETH
+-    '0x4e454152': 'eip155:60/slip44:60', // Aurora Mainnet (Ethereum L2 on NEAR) - Native symbol: ETH
+-    '0x63564c40': 'eip155:1666600000/slip44:1023', // Harmony Mainnet Shard 0 - Native symbol: ONE
++    '0x82750': null, //'eip155:534352/slip44:60', // Scroll Mainnet - Native symbol: ETH
++    '0x4e454152': null, //'eip155:60/slip44:60', // Aurora Mainnet (Ethereum L2 on NEAR) - Native symbol: ETH
++    '0x63564c40': null, //'eip155:1666600000/slip44:1023', // Harmony Mainnet Shard 0 - Native symbol: ONE
+ };
+ /**
+  * The list of chain IDs that can be supplied in the URL for the `/spot-prices`
+@@ -476,7 +478,7 @@ _CodefiTokenPricesServiceV2_policy = new WeakMap(), _CodefiTokenPricesServiceV2_
+         if (tokenAddresses.length === 0) {
+             return [];
+         }
+-        const url = new URL(`${BASE_URL_V2}/chains/${chainId}/spot-prices`);
++        const url = new URL(`${BASE_URL_V2}/chains/${parseInt(chainId, 16)}/spot-prices`);
```

Comment:
It works both with decimal and hexadecimal, but all the e2e tests are mocked using decimal, so this is easier than changing all of them.

When we remove the fallback to v2 they'll have to be changed anyway.